README.md

```diff
@@ -110,11 +114,25 @@ $ kubectl -n monitoring port-forward svc/testgrid-exporter 8081:8081
 
 You should be able to access the exposed metrics http://127.0.0.1:8081
 
-## Future plans
+## Roadmap
+
+- [X] Configure custom testgrid json exporter container
```

Comment:
we can remove the completed ones, sometime in the future

Reply:
Yes! Maybe after it's running in prod.